scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -0,0 +1,614 @@
+##! Module for recording and querying metrics. This modules wraps
+##! the lower-level telemetry.bif functions.
+##!
+##! Metrics will be exposed through a Prometheus HTTP endpoint when
+##! enabled by setting :zeek:see:`Broker::metrics_port` or using the
+##! `BROKER_METRICS_PORT` environment variable.
+
+@load base/misc/version
+
+module Telemetry;
+
+export {
+	## Alias for a vector of label values.
+	type labels_vector: vector of string;
+
+	## Type that captures options used to create metrics.
+	type MetricOpts: record {
```

Comment:
Maybe it's just me, but this name had me confused and I needed to look up the definition to figure out what it's actually about. So this is basically the definition of a metric, i.e., all the meta information without the actual values. So maybe `MetricDecl` or `MetricDef` would give a better intuition about the purpose of this type?

Reply:
This opts part comes from here: https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Opts

In the few usage examples and tests constructed, the name doesn't actually appear as it's an anonymous record, so leaving it for now.